src/content/guides/efk-stack-app-oidc/index.md

```diff
@@ -0,0 +1,365 @@
+What is missing?
+- [ ] Finish opendistro-security plugin
+    - [ ] Add overview
+    - [ ] Add explanation to `roles_key: groups` - this is what Kibana will use... 
+    - [ ] Identify other things that need explanation
+    - [ ] Explain them
+
+- [x] Add Configure internal-users database
+- [x] Add Configure roles-mapping to control users permission
+- [x] Add configuration of jwt token to send users group IDs
+- [x] Add explanation how azure AD roles are mapped to kibana roles
+
+---
+title: Configure oidc for efk-stack-app
```

Comment:
```suggestion
title: Configure OIDC for efk-stack-app
```

Reply:
Went through the hack.md one more time and just took it as is. Great fixes and improvements! Thanks so much.